app/main.ipynb

```diff
@@ -100,18 +125,30 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
-   "outputs": [],
    "source": [
-    "# write your code here"
+    "np_array = np.array(np.arange(2.0, 11.0))\n",
```

Comment:
The numpy array is created with values from 2.0 to 11.0, which results in a length of 9. Ensure the numbers are within the range from 0 to 10.0 as specified in the task.